.travis.yml

```diff
@@ -10,6 +10,8 @@ env:
   matrix:
     
     - CONDA_PY=27
+    - CONDA_PY=34
+    - CONDA_PY=35
```

Comment:
Hmm...not sure why these weren't here, but it does raise the question of whether they should be added. 😕